README.md

```diff
@@ -1,2 +1,91 @@
-# epic-flask-scaffold
-This is a scaffold for a React Flask app
+# Scaffold Setup Instructions
+
+This document outlines the setup instructions for both the backend and front-end components of the project. Ensure you follow the steps in sequence for a smooth setup.
+
+## Backend Setup in WSL
+
+### 1. Install Python 3.12.4
+Ensure Python 3.12.4 is installed in your WSL environment. Download it from the [official Python website](https://www.python.org/downloads/release/python-3124/).
+
+### 2. Set Up PYTHONPATH
+Add the following line to your `.bashrc` or `.zshrc` file to set the `PYTHONPATH` environment variable:
+export PYTHONPATH="/path/to/scaffold-api:${PYTHONPATH}"
+
+### 3. Configure Environment Variables
+Create a `.env` file in your scaffold-api with the necessary configurations. Replace placeholder values with actual secret keys where appropriate:
```

Comment:
not sure if we should add the .env values in the README 

thoughts? @dinesh-aot @saravanpa-aot

Reply:
sounds good ill remove it and reference sample.env